ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -404,6 +481,30 @@ export default class StatusDropdown extends React.Component {
               </React.Fragment>
             )}
           </DropdownMenu>
+          <div>
+            {alertStatus === 'untriaged' ? (
```

Comment:
Here I think we can replace `alertStatus === 'untriaged' ?` with `alertStatus === 'untriaged' &&`. It's cleaner imo since we get rid of the case in which we display `''`.